experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -0,0 +1,540 @@
+    canopy_model_args, canopy_component_args = create_canopy_args(FT, domain, earth_param_set, start_date)
+    canopy_component_types = (;
+        autotrophic_respiration = CL.Canopy.AutotrophicRespirationModel{FT},
+        radiative_transfer = CL.Canopy.TwoStreamModel{FT},
+        photosynthesis = CL.Canopy.FarquharModel{FT},
+        conductance = CL.Canopy.MedlynConductanceModel{FT},
+        hydraulics = CL.Canopy.PlantHydraulicsModel{FT},
+        energy = CL.Canopy.BigLeafEnergyModel{FT},
+    )
+    # setup snow model args
+    snow_args = create_snow_args(FT, domain, earth_param_set, dt)
+    snow_model_type = CL.Snow.SnowModel
+
+    # Setup overall land model
+    f_over = FT(3.28) # 1/m
+    R_sb = FT(1.484e-4 / 1000) # m/s
+    runoff_model =
+        CL.Soil.Runoff.TOPMODELRunoff{FT}(; f_over = f_over, f_max = spatially_varying_soil_params.f_max, R_sb = R_sb)
+
+    Csom = CL.PrescribedSoilOrganicCarbon{FT}(TimeVaryingInput((t) -> 5))
+
+    land_input = (
+        atmos = CL.CoupledAtmosphere{FT}(),
+        radiation = CL.CoupledRadiativeFluxes{FT}(),
+        runoff = runoff_model,
+        soil_organic_carbon = Csom,
+    )
+
+    model = CL.LandModel{FT}(;
+        soilco2_type = soilco2_type,
+        soilco2_args = soilco2_args,
+        land_args = land_input,
+        soil_model_type = soil_model_type,
+        soil_args = soil_args,
+        canopy_component_types = canopy_component_types,
+        canopy_component_args = canopy_component_args,
+        canopy_model_args = canopy_model_args,
+        snow_args = snow_args,
+        snow_model_type = snow_model_type,
+    )
+
+    Y, p, coords = CL.initialize(model)
+
+    # Set initial conditions
+    (; θ_r, ν) = spatially_varying_soil_params
+
+    # Apply temperature anomaly function to initial temperature
+    T_functions = Dict("aquaplanet" => temp_anomaly_aquaplanet, "amip" => temp_anomaly_amip)
+    haskey(T_functions, land_temperature_anomaly) ||
+        error("land temp anomaly function $land_temperature_anomaly not supported")
+    temp_anomaly = T_functions[land_temperature_anomaly]
+    T_sfc0 = FT(276.85) .+ temp_anomaly.(coords.subsurface)
+    lapse_rate = FT(6.5e-3)
+    # Adjust initial temperature to account for orography of the surface
+    # `surface_elevation` is a ClimaCore.Fields.Field(`half` level)
+    orog_adjusted_T_data = CC.Fields.field_values(T_sfc0) .- lapse_rate .* CC.Fields.field_values(surface_elevation)
+    orog_adjusted_T = CC.Fields.Field(orog_adjusted_T_data, subsurface_space)
+    orog_adjusted_T_surface = CC.Fields.Field(CC.Fields.level(orog_adjusted_T_data, 1), surface_space)
+
+    @. Y.soil.ϑ_l = θ_r + (ν - θ_r) / 2
+    Y.soil.θ_i .= FT(0.0)
+    ρc_s = CL.Soil.volumetric_heat_capacity.(Y.soil.ϑ_l, Y.soil.θ_i, soil_args.parameters.ρc_ds, earth_param_set)
+    Y.soil.ρe_int .= CL.Soil.volumetric_internal_energy.(Y.soil.θ_i, ρc_s, orog_adjusted_T, earth_param_set)
+    Y.soilco2.C .= FT(0.000412)
+
+    plant_ν = FT(1.44e-4)
```

Comment:
do you want to get this from the model to avoid defining 2x?
can access with land.canopy.hydraulics.parameters.plant_nu or \nu

Reply:
yes! I'm getting it from `canopy_component_args.hydraulics.parameters.ν` now